packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -31,7 +31,7 @@ export const DEFAULT_TEST_CONFIG = {
     locale: 'en-US',
     currency: 'USD',
     fetchedToken: 'test-token',
-    OCAPISessionsURL: `${DEFAULT_TEST_HOST}/mobify/proxy/ocapi/s/RefArch/dw/shop/v22_8/sessions`
+    OCAPISessionsURL: `${DEFAULT_TEST_HOST}/on/demandware.store/Sites-RefArch-Site/en_US/SLASSessionHelper-SyncHybridSession`
```

Comment:
This looks like the URL is pointing to a specific SLAS custom controller endpoint exposed by plugin_slas, is that correct? We don't want to introduce a dependency here on a specific version of plugin_slas. Mainly, we don't want to create a scenario where customer implementations are depending on custom controller logic that is changeable / deletable by customers since this is a footgun 🔫